PSReadLine/Cmdlets.cs

```diff
@@ -78,6 +89,10 @@ public class PSConsoleReadLineOptions
         public const ConsoleColor DefaultEmphasisColor  = ConsoleColor.Cyan;
         public const ConsoleColor DefaultErrorColor     = ConsoleColor.Red;
 
+        // Use dark black by default for the suggestion text.
+        // Find the most suitable color using https://stackoverflow.com/a/33206814
+        public const string DefaultPredictionColor = "\x1b[38;5;238m";
```

Comment:
See `LegacyWin32Console` which won't handle this color. Mapping arbitrary colors to a `ConsoleColor` seems problematic - it might be best to disable predictive display (at least as currently implemented) when using the `LegacyWin32Console`.

Reply:
Good point. I recall seeing issue reporting it doesn't work on 2012R2. How about throwing an error when setting `PredictionSource` to something other than `None` when `LegacyWin32Console` is in use?